### PR TITLE
Add NVMe NGUID info to volfact dict

### PR DESCRIPTION
**Pull request by sdodsley**
##### SUMMARY
Add NVMe NGUID value to response dict from module

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
purefa_volume.py

### CODE REVIEW

File: collections/ansible_collections/purestorage/flasharray/plugins/modules/purefa_info.py [hunk 1]
```diff
@@ -1043,7 +1045,7 @@ def generate_del_vol_dict(module, array):
             "created": vols[vol]["created"],
             "serial": vols[vol]["serial"],
             "page83_naa": PURE_OUI + vols[vol]["serial"],
-            "nvme_nguid": "00"
+            "nvme_nguid": "eui.00"
             + vols[vol]["serial"][0:14]
             + "24a937"
             + vols[vol]["serial"][-10:],
```
Comment (rdeenik) [inline collections/ansible_collections/purestorage/flasharray/plugins/modules/purefa_info.py:1051]: I believe this needs `.lower()` added, same as in `purefa_volume`.

File: collections/ansible_collections/purestorage/flasharray/plugins/modules/purefa_info.py [hunk 2]
```diff
@@ -1105,7 +1107,7 @@ def generate_vol_dict(module, array):
             "size": vols[vol]["size"],
             "serial": vols[vol]["serial"],
             "page83_naa": PURE_OUI + vols[vol]["serial"],
-            "nvme_nguid": "00"
+            "nvme_nguid": "eui.00"
             + vols[vol]["serial"][0:14]
             + "24a937"
             + vols[vol]["serial"][-10:],
```
Comment (rdeenik) [inline collections/ansible_collections/purestorage/flasharray/plugins/modules/purefa_info.py:1113]: Same here.

File: collections/ansible_collections/purestorage/flasharray/plugins/modules/purefa_info.py [hunk 3]
```diff
@@ -1157,7 +1159,7 @@ def generate_vol_dict(module, array):
                 "protocol_endpoint": True,
                 "source": vvols[vvol]["source"],
                 "serial": vvols[vvol]["serial"],
-                "nvme_nguid": "00"
+                "nvme_nguid": "eui.00"
                 + vols[vol]["serial"][0:14]
                 + "24a937"
                 + vols[vol]["serial"][-10:],
```
Comment (rdeenik) [inline collections/ansible_collections/purestorage/flasharray/plugins/modules/purefa_info.py:1165]: Same here.

**Review by rdeenik — CHANGES_REQUESTED** [review]
Please add `.lower()` for purefa_info. Otherwise all good.

**Comment by sdodsley** [comment]
@rdeenik lower added as requested - please recheck

**Review by rdeenik — APPROVED** [review]
shipit

**Pull request merged**